server/server.js

```diff
@@ -1,14 +1,216 @@
 const express = require("express");
 const cors = require("cors");
 const db = require("./db/db-connection.js");
-
-require("dotenv").config();
+require('dotenv').config()
+// console.log(process.env)
 const PORT = process.env.PORT || 3000;
 
 const app = express();
 app.use(cors());
 app.use(express.json());
 
+//SPECIES
+app.get('/species', async (req, res) => {
+    try {
+        const {rows : species} = await db.query('SELECT * FROM species');
+        res.send(species);
+    } catch (error) {
+        console.error("Error fetching species:", error.message);
+        res.status(500).send({ error: "Failed to fetch species." });
+    }
+});
+
+app.post('/species', async (req, res) => {
+    try {
+        const {common_name, scientific_name, estimated_number, conservation_code, created_at } = req.body;
+
+        const newSpecies = await db.query (
+            "INSERT INTO species (common_name, scientific_name, estimated_number, conservation_code, created_at) VALUES ($1, $2, $3, $4, $5) RETURNING *", [common_name, scientific_name, estimated_number, conservation_code, created_at]
```

Comment:
@beyzanurck Not sure if you all get this far but you should look at a ORM for DB queries. Makes life really ease with queries:

https://www.prisma.io/

But this still works as well.